### PR TITLE
Merge release into develop

### PR DESCRIPTION
**Pull request by Wovchena**

### CODE REVIEW

File: tools/accuracy_checker/accuracy_checker/main.py [hunk 1]
```diff
@@ -437,6 +437,7 @@ def main():
         except Exception as e:  # pylint:disable=W0703
             details['status'] = 'error'
             details['error'] = str(type(e))
+            send_telemetry_event(tm, 'status', 'failure')
             send_telemetry_event(tm, 'model_run', json.dumps(details))
             send_telemetry_event(tm, 'status', 'failure')
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/main.py:442]:
```suggestion
```

File: tools/accuracy_checker/accuracy_checker/annotation_converters/convert.py [hunk 2]
```diff
@@ -1,12 +1,9 @@
 """
 Copyright (c) 2018-2021 Intel Corporation
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/annotation_converters/convert.py:2]:
```suggestion
Copyright (c) 2018-2021 Intel Corporation

```

File: tools/accuracy_checker/accuracy_checker/annotation_converters/convert.py [hunk 3]
```diff
@@ -1,12 +1,9 @@
...
-
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/annotation_converters/convert.py:5]:
```suggestion
You may obtain a copy of the License at

```

File: tools/accuracy_checker/accuracy_checker/annotation_converters/convert.py [hunk 4]
```diff
@@ -1,12 +1,9 @@
...
-
       http://www.apache.org/licenses/LICENSE-2.0
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/annotation_converters/convert.py:6]:
```suggestion
      http://www.apache.org/licenses/LICENSE-2.0
     
```

File: tools/accuracy_checker/accuracy_checker/annotation_converters/convert.py [hunk 5]
```diff
@@ -6,7 +6,7 @@
 You may obtain a copy of the License at
 
       http://www.apache.org/licenses/LICENSE-2.0
-
+
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/annotation_converters/convert.py:9]:
```suggestion

```

**Review by eaidova — APPROVED** [review]

**Comment by Wovchena** [comment]
I can see why there were line removals and two `send_telemetry_event(tm, 'status', 'failure')`. The lines are removed only in release. `send_telemetry_event(tm, 'status', 'failure')` was misplaced in one of brunches and thus didn't conflict. But why are there two `ANCHORS` in demos/common/python/models/yolo.py? There wasn't any conflict on that. GIt just added them. @eaidova, @eizamaliev, any idea?

**Comment by Wovchena** [comment]
Why does git even show these as changes of a merge commit if they don't conflict?

> [the `git log` for any merge to see how something was resolved after the fact](https://git-scm.com/book/en/v2/Git-Tools-Advanced-Merging)

**Review by vladimir-dudnik — APPROVED** [review]

**Comment by Wovchena** [comment]
Thanks to @IRDonch, there is a possible explanation. The line above anchors was different:
`from .utils import Detection, resize_image, resize_image_letterbox, load_labels` and
`from .utils import Detection, resize_image, resize_image_letterbox, load_labels, clip_detections`. That probably prevented git from matching `ANCHORS`. At the same time when merging, it was possible to add second `ANCHORS` looking at surroundings: empty line on top and `class YOLO(Model):` below.


The book is misleading on
> [the `git log` for any merge to see how something was resolved after the fact](https://git-scm.com/book/en/v2/Git-Tools-Advanced-Merging)

The explanation which is more aligned with what `git show d034584` does is
> [compresses the patch output by omitting uninteresting hunks whose the contents in the parents have only two variants and the merge result picks one of them without modification](https://git-scm.com/docs/git-diff-tree#Documentation/git-diff-tree.txt---cc)

**Pull request merged**